src/ssh.c

```diff
@@ -205,8 +219,13 @@ int dcc_ssh_connect(char *ssh_cmd,
     if (!path)
         path = (char *) "distccd";
 
+    char *child_argv[10+num_ssh_args];
```

Comment:
I moved it so we could allocate the correct size (although I should have used the heap for this if I really wanted to do that.)  I would only favor moving it back if we set a larger max and check our bounds as you requested in the note at line 208.  I can do that.